docs/content/docs/dev/table/tableApi.md

```diff
@@ -2735,6 +2735,114 @@ result = t.select(col('a'), col('c')) \
 
 {{< query_state_warning >}}
 
+### Model Inference
+
+{{< label Streaming >}}
+
+The Table API supports model inference operations that allow you to integrate machine learning models directly into your data processing pipelines. You can create models with specific providers (like OpenAI) and use them to make predictions on your data.
```

Comment:
As we use the words inference and predict, I think we should be explicit about what we mean. Google is saying "Model inference is about understanding the underlying relationships and mechanisms in data to make conclusions about existing data, while prediction is about forecasting future outcomes or estimating probabilities for new data points". What do we mean?

Reply:
I can change it to inference